packages/protect/package.json

```
@@ -53,7 +53,7 @@
 	},
 	"dependencies": {
 		"@byteslice/result": "^0.2.0",
-		"@cipherstash/protect-ffi": "0.14.2",
+		"@cipherstash/protect-ffi": "0.15.0-0",
```

```suggestion
		"@cipherstash/protect-ffi": "0.15.0",
```

Heads up that I just released `0.15.0` of `protect-ffi`, so you don't need to use the prerelease anymore.